Formula/open-ocd.rb

```diff
@@ -11,7 +11,6 @@ class OpenOcd < Formula
   end
 
   bottle do
-    rebuild 2
```

Comment:
Please don't touch the `bottle` block. Squash your commits together when you revert this. Thanks!